src/bindings/python/src/pyopenvino/core/properties/properties.cpp

```diff
@@ -106,11 +106,16 @@ void regmodule_properties(py::module m) {
     py::module m_intel_cpu =
         m_properties.def_submodule("intel_cpu", "openvino.properties.intel_cpu submodule that simulates ov::intel_cpu");
 
+    py::enum_<ov::intel_cpu::TbbPartitioner>(m_intel_cpu, "TbbPartitioner", py::arithmetic())
```

Comment:
Please, add direct import also [here](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/src/openvino/properties/intel_cpu/__init__.py)
[for example](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/src/openvino/properties/intel_auto/__init__.py) other enum in other submodule